UnityProject/Assets/Scripts/Clothing/Eyewear/NightVisionGoggles.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+using CameraEffects;
+
+public class NightVisionGoggles : NetworkBehaviour, IServerInventoryMove
+{
+	[SerializeField, Tooltip("How far the player will be able to see in the dark while he has the goggles on.")]
+	private Vector3 nightVisionVisibility;
+
+	[SerializeField, Tooltip("The default minimal visibility size.")]
+	private Vector3 defaultVisionVisibility;
+
+	
+	private void OnStartClient()
+	{
+		if(PlayerManager.LocalPlayerScript != null)
+		{
+			var item = PlayerManager.LocalPlayerScript.Equipment.GetClothingItem(NamedSlot.eyes).GameObjectReference;
+			if(item == gameObject) {enableEffect(true);}
+		}
+	}
```

Comment:
Also add a note here saying this wont work as equipment wont be sync'd in time for the check, and that IServerInventoryMove should be replaced with IClienInventoryMove but that needs more functionality first